### DISCUSSION TITLE
输出面板OutputService的样式问题

### DISCUSSION THREAD

**Discussion by chaoyue1217** [opensumi/core#2402] [Q&A]
![企业微信截图_16786152429461](https://user-images.githubusercontent.com/18662948/224641157-482f9726-57c4-477c-aa8f-5f5b6c32e44a.png)
opensumi的输出面板是否可实现如上图vscode中输出所示的，点击一个路径打开文件的功能？

**Comment by erha19** [comment]
@chaoyue1217  对于这部分路径的识别，在 OpenSumi 内有一个逻辑在处理，这里应该还是部分情况没有正确识别到链接，可以看一下这里的代码看能不能调试修复一下，或等待我们后续排查一下 

https://github.com/opensumi/core/blob/9af5e4fbc53bbb3d76ce88b88b07817d09b941fd/packages/output/src/browser/output-link.provider.ts#L38~L81

**Reply by chaoyue1217** [reply]
按照您给的地址调试过了，输出的正则和我写的路径是匹配的上的，但是还是没有显示出来链接，是否需要什么插件或者是开启什么配置吗？
